runtime/compiler/z/codegen/CodeGenPhaseToPerform.hpp

```diff
@@ -59,6 +59,7 @@
     RegisterAssigningPhase,
     MapStackPhase,
     PeepholePhase,
+    ExpandInstructionsPhase,
```

Comment:
Why do we add this to the Z codegen phases list vs just into the common list or p list as well?

Reply:
I think I see: only z has its own list.

Reply:
Figured it out.